plugins/accuracy.h

```diff
@@ -28,7 +28,7 @@ struct AccuracyConfig : public ProcedureConfig {
     static constexpr const char * name = "classifier.test";
 
     AccuracyConfig()
-          : mode(CM_BOOLEAN), uniqueScoresOnly(false)
+          : mode(CM_BOOLEAN), uniqueScoresOnly(false), accuracyOverN(1)
```

Comment:
Why is that? All of the plugins config classes initialize the defaults in the constructor.